### DISCUSSION TITLE
Kafka MirrorMaker2 not connecting to Azure Eventhub

### DISCUSSION THREAD

**Discussion by minimax75** [strimzi/strimzi-kafka-operator#7352] [Q&A] [answered]
Hello,

Please help me sort out a problem. I'm trying to mirror a topic from an on premises Strimzi Kafka cluster to Azure Eventhub.
I am running the Strimzi operator (0.30.0) in a Kubernetes cluster behind a proxy. I have followed [this guide/blog](https://strimzi.io/blog/2020/06/09/mirror-maker-2-eventhub/) to setup MirrorMaker2 and set up the https_proxy and http_proxy env variables in the strimzi operator deployment yaml following [this guide](https://strimzi.io/docs/operators/latest/configuring.html#proc-configuring-proxy-config-cluster-operator-str).

However, when launching the MirrorMaker2 a connection to Azue Eventhub is not possible an the pod restarts.
[cluster-to-eh-mirrormaker2-error.log](https://github.com/strimzi/strimzi-kafka-operator/files/9605367/cluster-to-eh-mirrormaker2-error.log)

MirrorMaker2 definition:
[mirrormaker2.txt](https://github.com/strimzi/strimzi-kafka-operator/files/9605432/mirrormaker2.txt)

Any ideas for a fix?

Thank you!

**Comment by scholzj** [comment, marked as answer]
The Kafka connections are not HTTP based. So I don't think setting the HTTP/HTTPS proxy will help you in any way.

**Reply by minimax75** [reply]
Many Thanks Jakub,

for the hint, this was the missing link. After setting up the routing, connection could be estblished.
